src/mpid/ch4/netmod/ofi/errnames.txt

```diff
@@ -107,3 +107,5 @@
 **ofi_max_conn %d:OFI dynamic process reaches maximum connection (%d)
 **ofi_num_nics:OFI has a different number of nics available on different notes
 **ofi_num_nics %d %d:OFI has a different number of nics available on different notes (%d on this node and %d elsewhere)
+**ofid_mr_key:OFI MR key invalid
```

Comment:
Remove the next one (line 111) since it is not used.

Reply:
Done.